tests/component-tests/textfieldSlider/js/TextfieldSliderTests.js

```diff
@@ -99,8 +183,26 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                     value: 1
                 }
             });
-            fluid.tests.textfieldSlider.testInputField("aaa", 1);
-            fluid.tests.textfieldSlider.testInputField(null, 0);
+            fluid.tests.textfieldSlider.testInputField("aaa", 1, that);
+            fluid.tests.textfieldSlider.testInputField(null, 0, that);
         });
+
+        jqUnit.test("Test Invalid Values (jQuery)", function () {
+            jqUnit.expect(4);
+
+            var that = fluid.tests.textfieldSlider.createTextfieldSliderJQuery({
+                range: {
+                    min: -5,
+                    max: 5
+                },
+                model: {
+                    value: 1
+                }
+            });
+            fluid.tests.textfieldSlider.testInputField("aaa", 1, that);
+            fluid.tests.textfieldSlider.testInputField(null, 0, that);
+        });
```

Comment:
These two tests seem quite duplicated with the primary difference being which version of the slider to use. Maybe parameterizing this would help.